public/setRequestList.php

```diff
@@ -61,6 +61,7 @@
             echo "<td><select class='gameselector' onchange='window.location = \"/setRequestList.php\" + this.options[this.selectedIndex].value'><option value=''>-- All Systems --</option>";
 
             foreach ($consoles as $console) {
+                sanitize_outputs($console['Name']);
```

Comment:
"Taint analysis" (I keep a console with a script tag in the title around ;) suggested that the console name should be sanitized in the dropdown.